doc/admin/deployment.md

```diff
@@ -870,24 +872,26 @@ If you wish to use systemd with a development build, you must copy the service
 file from utils/systemd to /usr/lib/systemd/system. Once the file is copied
 modify the ExecStart line to point to your in tree daos_agent binary.
 
+ExecStart=/usr/bin/daos_agent -i -o <'path to agent configuration file/daos_agent.yml'>
```

Comment:
0.9 has full working certificate support should we update this to remove the -i?